src/members/models/member.py

```diff
@@ -184,33 +183,41 @@ def roles(self):
             positions__applications__status='appointed',
         )
 
+    @property
+    def get_status(self):
+        self.update_status()
+        return self.status
+
     def person_number(self) -> str:
         if self.birthday is None or self.person_number_ext is None:
             return ''
         else:
             return '%s-%s' % (self.birthday.strftime('%Y%m%d'),
                               self.person_number_ext)
 
-    def update_status(self, data=None):
+    def update_status(self, data=None, save=True):
         if data is None:
-            if self.person_number() == '':
+            # Prevent updating this value to often
+            if timezone.now() - self.status_changed >= timedelta(days=1):
```

Comment:
timdelta should be updated to days=3

Reply:
I believe days=1 is correct, right? 
The cron job only calls this function every 3 days, but once this function gets called, it will update the status for all members that haven't been updated in the last 24 hours

Reply:
What Bjoskim said ☝️
Because the cronjob runs an update for each member in the database, we want to restrict as many as possible.
The cronjob already filters out the ones that hasn't been updated the last 3 days. There are other parts of the code where this function is called, and because this is just for a single member it's ok to make these requests each 24 hour.

Reply:
Bjoskim 🥇